refdb-sql/.gitignore

```diff
@@ -0,0 +1,2 @@
+*~
+*.sql.org.org.org.org.org.org.org
```

Comment:
Is this really supposed to be here, it seems like this file could be removed and superseded by the top level gitignore.